README.md

```diff
@@ -6,49 +6,7 @@
 
 ![Arcus](https://raw.githubusercontent.com/arcus-azure/arcus/master/media/arcus.png)
 
-# Installation
-
-## Web API Template
-We provide a template to build Web APIs. ([docs](https://templates.arcus-azure.net/features/web-api-template))
```

Comment:
The advantage of this was, that you could see this info immediately when opening the project on github, without having to look into the separate docs.
But I can agree that it is duplicate info and therefore also hard to maintain

Reply:
Yes, that's the reason for this proposal. I think that even all `README` files were out-of-date.